src/zac/core/forms.py

```diff
@@ -323,37 +323,95 @@ class SelectUsersReviewRequestForm(forms.Form):
         help_text=_("Select the advisors."),
     )
 
+    deadline = forms.DateField(
+        required=True,
+        label=_("Deadline"),
+        help_text=_("Select a date"),
+        input_formats=["%Y-%m-%d"],
+    )
+
 
 class BaseReviewRequestFormSet(BaseTaskFormSet):
-    def is_valid(self):
-        super().is_valid()
+    def deadlines_validation(self) -> bool:
+        """
+        Validate that deadlines per step are monotonic increasing
+        """
+        valid = True
+        deadline_old = date(1, 1, 1)
         for form in self.forms:
-            if not form.has_changed():
-                form.add_error(None, _("Please select at least 1 advisor."))
-                return False
+            deadline_new = form.cleaned_data["deadline"]
+            if deadline_new and not deadline_new > deadline_old:
+                form.add_error(
+                    None,
```

Comment:
```suggestion
                    "deadline",
```
You can attach the error directly to the form.deadline field instead of putting it in the `form.non_field_errors`